kubernetes/kubernetes.go

```diff
@@ -443,3 +454,19 @@ func NewKubernetesJobClient(name string, namespace string) (*KubernetesJobClient
 	}
 	return &KubernetesJobClient{Clientset: clientset, JobName: name, Namespace: namespace}, nil
 }
+
+func getConcurrencyPolicy() (batchv1.ConcurrencyPolicy, error) {
+	concurrencyPolicyEnv := helpers.GetEnv("JOBS_CONCURRENCY_POLICY", "Allow")
```

Comment:
Options we have

// Specifies how to treat concurrent executions of a Job.
// Valid values are:
// - "Allow" (default): allows CronJobs to run concurrently;
// - "Forbid": forbids concurrent runs, skipping next run if previous run hasn't finished yet;
// - "Replace": cancels currently running job and replaces it with a new one
// +optional
The forbid might be useful if you schedule a job every minute and it does a shit ton of compute making it take 4eva. With the current default, you have a ton queuing and it could OOM your cluster. I'm adding it as an env var so we can config it for local tests